Improve list of dependencies required for installation

Downloaded Lyra and ran make. After getting through the download of all the dependent projects (15+ minutes on my mifi connection), I encountered this error:

🔘 Ensuring node version is 9.0.0 or later (12:31:31)
🔴 must be running Node version 9.0.0 or later.  Exiting
make: *** [check-node] Error 1

**Describe the solution you'd like**
Would be super helpful to do a dependency check up front or list in the docs what version of Node is required.

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,7 +12,7 @@ jobs:
         - nvm install 11
         - nvm use 11
         - node --version
-        - make all
+        - make all smoke-test-ts
       before_deploy:
         - sudo apt-get install -y upx
         - make dist-release

--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ all: TESTFLAGS = --race
 all: everything
 
 PHONY+= everything
-everything: check-mods clean lint test lyra plugins smoke-test smoke-test-ts
+everything: check-mods clean lint test lyra plugins smoke-test
 
 PHONY+= shrink
 shrink:

--- a/README.md
+++ b/README.md
@@ -28,6 +28,7 @@ The project requires [Go](https://golang.org/doc/install) 1.11 or higher, and [g
 
 1. Clone this repository: `git clone https://github.com/lyraproj/lyra`
 2. Build Lyra: `cd lyra; make`
+3. Optionally, if you intend to work with typescript, run `make smoke-test-ts` (this will check for an appropriate version of Node.js)
 
 ### Deploying Workflows with the Lyra CLI
 

